### DISCUSSION TITLE
What is the roadmap for Keystatic? (2024)

### DISCUSSION THREAD

**Discussion by tyteen4a03** [Thinkmill/keystatic#877] [General]
What is the roadmap for Keystatic? I am seeing a bunch of PRs being merged in that seems to have happened out of thin air, and there are a good chunk of issues / discussions that have received no attention.

**Comment by simonswiss** [comment]
Hey!

A majority of the Keystatic core team has been either on holiday or working on other projects in the end of 2023 / start of 2024. We'll do our best to get back on top of issues/discussions.

I'm afraid there is no obvious concrete roadmap, but here's a little summary of the things coming up to Keystatic:

- New editor with extended capabilities for content components, bring life to `wrapper`, `block`, `inline`, `mark` and `repeatable` components — these should allow you to create really advanced content editing experiences
- Support for MDX 🔥 The new (experimental for now) MDX field works just like the `document` field, but... with MDX.

Hope it helps!

**Comment by airtonix** [comment]
> What is the roadmap for Keystatic? I am seeing a bunch of PRs being merged in that seems to have happened out of thin air, and there are a good chunk of issues / discussions that have received no attention.

I imagine this is because Thinkmill have to quantify the time they spend on working on Keystatic, and as such would most likely be doing a lot of internal planning on a issue tracker that already fits in with their internal reporting tools.

As such, "out of thin air" is probably just them working from their private jira board (or something like it).
